resolver/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.2"
```

Comment:
I actually don't even know why i this is needed. I just put it here because the example poetry project from `poetry new` command creates a project containing it.  I changed to use importlib.metadata as you suggested but if it makes sense, I wouldn't mind just getting rid of the version here and only rely on version in pyproject.toml file.  Should I just remove it @tarkatronic ?

Reply:
Oh, yeah, if the code doesn't use this then I would just get rid of it for sure!

Reply:
removed.  Thanks @tarkatronic !